### PR TITLE
feat(cleanup): remove !important part 1

### PR DESCRIPTION
**Pull request by cpooley**
Part 1 of removing !important

### CODE REVIEW

**Comment by cpooley** [comment]
The snapshot test for Footer, shows h3 as 3rem and h4 as 2.125rem, so it's taking it from mui and not our theme. What's the best way to change that?

**Comment by shekhar-shubhendu** [comment]
> The snapshot test for Footer, shows h3 as 3rem and h4 as 2.125rem, so it's taking it from mui and not our theme. What's the best way to change that?

if you create a WrappedComponent with ThemeProvider set then it will work
example: https://github.com/tzConnectBerlin/prediction-market-ui/blob/d4c05adce0fb926b866d5ea271abbbeb6f77aef0/src/design-system/molecules/FormikTextField/FormikTextField.test.tsx#L80

**Review by tlebon — APPROVED** [review]

**Review by shekhar-shubhendu — APPROVED** [review]
LGTM

Feel free to merge. :)

File: src/design-system/atoms/Button/Button.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ export const CustomButton: React.FC<ButtonProps> = ({
       endIcon={iconPosition === 'right' ? icon : null}
       bordercolor={internalBorderColor}
       texttype={lowercase ? 'none' : 'uppercase'}
-      sx={{ ...customStyle }}
+      sx={customStyle}
```
Comment (shmodarresi) [inline src/design-system/atoms/Button/Button.tsx:87]: Please remove **customStyle** of props and just use **sx**?

**Review by shmodarresi — CHANGES_REQUESTED** [review]

File: src/design-system/atoms/Button/Button.tsx (hunk 1 above)
Comment (cpooley) [reply src/design-system/atoms/Button/Button.tsx:87]: My last pr was about removing sx :D 
Does it cause problems?

**Review by shmodarresi — APPROVED** [review]

**Pull request merged**